easybuild/easyblocks/t/tensorflow.py

```diff
@@ -1136,10 +1141,16 @@ def install_step(self):
             whl_version = self.version.replace("-rc", "rc")
         else:
             whl_version = self.version
-
-        whl_paths = glob.glob(os.path.join(self.builddir, 'tensorflow-%s-*.whl' % whl_version))
-        if not whl_paths:
-            whl_paths = glob.glob(os.path.join(self.builddir, 'tensorflow-*.whl'))
+        if LooseVersion(self.version) < LooseVersion('2.16'):
+            whl_paths = glob.glob(os.path.join(self.builddir, f"tensorflow-{whl_version}-*.whl"))
+            if not whl_paths:
+                whl_paths = glob.glob(os.path.join(self.builddir, 'tensorflow-*.whl'))
+        else:
+            whl_paths = glob.glob(os.path.join(
+                self.builddir,
+                f'TensorFlow/tensorflow-{self.version}/bazel-bin/tensorflow/tools/pip_package/wheel_house',
+                f"tensorflow-{whl_version}-*.whl"
```

Comment:
The other code path has a fallback. Not sure how relevant this still is but maybe we should keep that.
Idea: set `whl_dir = self.builddir` if <2.16 else `that-long-thing` and then use the existing code with the fallback with whl_dir instead of builddir